bftengine/include/bcstatetransfer/SimpleBCStateTransfer.hpp

```diff
@@ -92,11 +93,6 @@ class IAppState {
   // and getLastBlockNum() should always return the same block number.
   // When that state transfer module is updating the state, then these methods
   // may return different block numbers.
-
```

Comment:
No more wait() method?

Reply:
No, in order to keep replica responsive even when the object store is temporarily inaccessible, we handoff state transfer messages to another dedicated thread and it waits synchronously for the object store

Reply:
It works only in RO replica, which doesn't maintain its own state. So there's no thread safety problems